saw-script/src/SAWScript/Typechecker.hs

```diff
@@ -1472,6 +1475,15 @@ inferStmt cname atSyntacticTopLevel blockpos ctx s =
             return s
         StmtImport _spos _ ->
             return s
+        StmtInclude spos _ -> do
+            -- Restrict include to TopLevel. This matches the prior
+            -- behavior when it was a builtin function rather than
+            -- syntax. FUTURE: consider relaxing the requirement.
```

Comment:
It seems like this restriction doesn't actually apply, given the documented and tested use of `include` in `ProofScript`...